Source/VSProps/Base.props

```diff
@@ -111,6 +111,12 @@
                 Currently jits use some annoying code patterns which makes this common
       -->
       <DisableSpecificWarnings>4245;%(DisableSpecificWarnings)</DisableSpecificWarnings>
+      <!-- Disable warnings that get tripped when the optimizer gets too smart for it's own good.
+      For instance, it thinks a value can be zero, but actually the value is set from an API like OpenGL
+      which specifies bounds. Technically the compiler is correct, but...
+      4724  potential mod/div by 0
+      -->
+      <DisableSpecificWarnings Condition="'$(DolphinRelease)'=='true'">4724;%(DisableSpecificWarnings)</DisableSpecificWarnings>
```

Comment:
fwiw, this complains about `s_ubo_align` in ProgramShaderCache.cpp , specifically when passed to `Common::AlignUp` and `StreamBuffer::Map`, which mod by `s_ubo_align`.